src/proxy-ui-api/src/test/java/org/niis/xroad/securityserver/restapi/service/ClientServiceIntegrationTest.java

```diff
@@ -809,94 +877,221 @@ public void findLocalClientsByAllTermsExcludeMembers() {
     }
 
     @Test
-    public void findLocalClientsByOnlyLocalClientsWithValidSignCert() throws Exception {
-        when(currentSecurityServerSignCertificates.getSignCertificateInfos()).thenReturn(createCertificateInfoList());
+    public void findClientsByHasValidLocalSignCertSimpleScenario() throws Exception {
```

Comment:
`throws Exception` could be removed

Reply:
fixed in https://github.com/nordic-institute/X-Road/commit/4a96015f6a94a041e0f1484b01fca9735ccd18ed